source/imaer-gml/src/main/java/nl/overheid/aerius/gml/base/source/mobile/v31/GML2OffRoad.java

```diff
@@ -99,7 +100,7 @@ private void convert(final T source, final IsGmlCustomOffRoadMobileSource custom
     newSource.setSectorId(sectorId);
     newSource.setLabel(constructLabel(source.getLabel(), customMobileSource.getDescription()));
     newSource.setCharacteristics(gml2SourceCharacteristics.fromGML(customMobileSource.getCharacteristics(),
-        getConversionData().determineDefaultCharacteristicsBySectorId(sectorId), null));
+        getConversionData().determineDefaultCharacteristicsBySectorId(sectorId, GeometryType.POINT), null));
```

Comment:
Wouldn't it be safer to use the geometry as set on line 110 below instead use `POINT`?

Reply:
Good point, will change it